neurodsp/tests/conftest.py

```diff
@@ -22,3 +25,8 @@ def tsig():
 def tsig2d():
 
     yield np.random.randn(2, 1000)
+
+@pytest.fixture(scope='session')
+def tsig_sine():
+
+	yield sim_oscillation(N_SECONDS, FS, freq=FREQ_SINE, variance=None, mean=None)
```

Comment:
I updated this to use general default test parameters, and so not package and return this data. 

I like this in part because any test using this signal can use the settings value to know things like FS, without needing all the unpacking code.